### PR TITLE
Added bonk feature issue #89

### PR DESCRIPTION
**Pull request by king2b3**
Issue #89 
Dad will now BONK someone that is being naughty
A random bonk gif will be returned if someone says something dad finds distasteful
It would be funny to one day have dad jump in a voice chat with you and play the bonk sound if you say something bad in a text channel and are currently in a voice channel, but I'll open an issue for that later on.

### CODE REVIEW

**Comment by king2b3** [comment]
BTW dracarys will need to install the library.
pip3 install profanity-check

**Comment by I-question-this** [comment]
@king2b3 That requirement should be added via ``requirements.txt``

**Comment by king2b3** [comment]
Gotcha, let me add that real quick

File: jokes/bonk.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        predict([msg]) Returns
+        ----------------------
+        True
+            Message is profain
+        False
+            Message is clean
+        """
+        
+        # Set up super class
+        super().__init__("bonk", 100.0)
```
Comment (I-question-this) [inline jokes/bonk.py:31]: I recommend having the default be significantly less than 100% as people tend to swear in chats quite frequently for benign reasons. I would recommend 20% as max, but maybe lower like 10% or 5%.

File: requirements.txt [hunk 2]
```diff
@@ -1,2 +1,3 @@
 # These are the main requirements, but they have several other dependencies
 Red-DiscordBot==3.4.0
+profanity-chec==1.0.3
```
Comment (I-question-this) [inline requirements.txt:3]: Misspelled module name.

Comment (I-question-this) [reply requirements.txt]: Resolved this myself when I updated the minimum requirement for Red-DiscordBot as it caused a merge conflict.

**Comment by I-question-this** [comment]
@king2b3 Any idea on how to resolve this error with the profanity_check package? (Yes I did install the package via `pip install -r requirements.txt` Attempting to install joblib via `pip install joblib` reports that joblib version 0.17.0 is already installed.

```
[2020-11-08 22:13:40] [ERROR] red: Package loading failed
Traceback (most recent call last):
  File "/home/tyler/redenv/lib/python3.8/site-packages/redbot/core/core_commands.py", line 171, in _load
    await bot.load_extension(spec)
  File "/home/tyler/redenv/lib/python3.8/site-packages/redbot/core/bot.py", line 1202, in load_extension
    lib = spec.loader.load_module()
  File "<frozen importlib._bootstrap_external>", line 462, in _check_name_wrapper
  File "<frozen importlib._bootstrap_external>", line 962, in load_module
  File "<frozen importlib._bootstrap_external>", line 787, in load_module
  File "<frozen importlib._bootstrap>", line 265, in _load_module_shim
  File "<frozen importlib._bootstrap>", line 702, in _load
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/tyler/Development/Python/Red-DiscordBot_Cogs/dad/__init__.py", line 1, in <module>
    from .jokes import JOKES
  File "/home/tyler/Development/Python/Red-DiscordBot_Cogs/dad/jokes/__init__.py", line 7, in <module>
    from .bonk import BonkJoke
  File "/home/tyler/Development/Python/Red-DiscordBot_Cogs/dad/jokes/bonk.py", line 4, in <module>
    from profanity_check import predict
  File "/home/tyler/redenv/lib/python3.8/site-packages/profanity_check/__init__.py", line 1, in <module>
    from .profanity_check import predict, predict_prob
  File "/home/tyler/redenv/lib/python3.8/site-packages/profanity_check/profanity_check.py", line 3, in <module>
    from sklearn.externals import joblib
ImportError: cannot import name 'joblib' from 'sklearn.externals' (/home/tyler/redenv/lib/python3.8/site-packages/sklearn/externals/__init__.py)
[2020-11-08 22:14:16] [INFO] red.main: Signals.SIGINT received. Quitting...
[2020-11-08 22:14:16] [INFO] red.main: Shutting down with exit code: ExitCodes.SHUTDOWN
[2020-11-08 22:14:16] [INFO] red.main: Signals.SIGTERM received. Quitting...
[2020-11-08 22:14:16] [INFO] red.main: Please wait, cleaning up a bit more
```

**Comment by king2b3** [comment]
Hm I'm not sure, I didn't run into that yet. I'll check it out tomorrow

**Comment by I-question-this** [comment]
As it currently stands [profanity-check](https://pypi.org/project/profanity-check/)'s build is failing, which is the error we are seeing. Once they figure out their problem we can hopefully make use of it to implement our code.

**Comment by I-question-this** [comment]
Combining a response to "bonk" and profanity is a bad idea. "bonk" is rarely used in conversation so the probability of responding should be high. Swearing is commonly used frequently, so the probability of responding should be low.

Thus, please readd the module for the initial idea and separate these ideas into to different Joke classes.

**Review by I-question-this — APPROVED** [review]
It all looks dandy to me.

**Pull request merged**